4-2.js

```diff
@@ -0,0 +1,11 @@
+class Animal {
+  constructor(move) {
+      this.move = move;
+      console.log(this.move)
+  }
+}
+
+class Cat extends Animal {
+ }
+let cat = new Cat('run');
+cat.move
```

Comment:
console.log(cat.move)
del console.log from constructor
and it's better to create constructor in Cat class even if it's default